action.yml

```diff
@@ -59,13 +60,31 @@ runs:
       uses: ublue-os/remove-unwanted-software@v6
       if: ${{ inputs.maximize_build_space == 'true' }}
 
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+      with:
+        install: true
+        driver: docker
+
+    - name: Install BlueBuild
+      shell: bash
+      env:
+        # Uses GitHubs ternary syntax to set cli version, see https://docs.github.com/en/actions/learn-github-actions/expressions#example
+        CLI_VERSION_TAG: ${{ inputs.use_unstable_cli == 'true' && 'main' || 'v0.8.2' }}
+      run: |
+        docker run \
+          --detach \
+          --rm \
+          --name blue-build-installer \
+          ghcr.io/blue-build/cli:${{ env.CLI_VERSION_TAG }}-installer \
```

Comment:
Is this actually the correct syntax for env here? I thought that `env:` adds to the bash env, and you would have to access those variables through the bash env syntax. Apologies if you tested this already and it worked.

Reply:
Yeah you can do either

https://docs.github.com/en/actions/learn-github-actions/variables#using-the-env-context-to-access-environment-variable-values